tests/app/api/services/test_suppliers.py

```diff
@@ -27,10 +27,6 @@ def supplier(self, app, request):
                 'qld': {
                     'expiry': expiry,
                     'licenceNumber': 'Q123456'
-                },
-                'sa': {
-                    'expiry': expiry,
-                    'licenceNumber': 'S123456'
```

Comment:
Should we leave this in here and write a test with an expired SA licence?  What about writing tests to check the application and supplier validators when there's no data or partial data for SA?